modules/ding_user/ding_user.admin.inc

```diff
@@ -17,14 +17,3 @@ function ding_user_admin_setting_form($form, &$form_state) {
 
   return system_settings_form($form);
 }
-
-/**
- * Validate user admin form.
- */
-function ding_user_admin_setting_form_validate($form, &$form_state) {
-  if (!empty($form_state['values']['ding_user_signup_link'])) {
-    if (!(0 === strpos($form_state['values']['ding_user_signup_link'], 'http'))) {
-      form_set_error('ding_user_signup_link', t('User signup link must start with "http"'));
-    }
-  }
```

Comment:
Disabling validation entirely to allow for relative urls seems a bit harsh. How about using [`valid_url()`](https://api.drupal.org/api/drupal/includes%21common.inc/function/valid_url/7.x) instead?

```php
TRUE == valid_url('/gatewayf/registration/information');
```